lib/sinatra/base.rb

```diff
@@ -81,7 +81,7 @@ def params
     end
 
     class AcceptEntry
-      attr_accessor :params
+      attr_reader :params
```

Comment:
Could be a breaking change?

Reply:
Checked the code and it looks AcceptEntry.params wont get modified anywhere